delocate/delocating.py

```diff
@@ -56,6 +58,12 @@ class DelocationError(Exception):
     pass
 
 
+def posix_relpath(path: str, start: str = None) -> str:
+    """Return path relative to start using posix separators (/)."""
+    rel = relpath(path, start)
+    return Path(rel).as_posix()
```

Comment:
If the goal is cross platform portability then do not use `str` for path types in new functions.